src/app/frontend/deploy/createsecret.html

```diff
@@ -17,7 +17,8 @@
 <md-dialog aria-label="Create a new secret" layout="column">
   <md-dialog-content layout-padding>
     <h4 class="md-title">Create a new image pull secret</h4>
-    <div>The new secret will be added to the cluster.</div>
+    <div>The new secret will be added to the cluster.</p>
```

Comment:
It should be `<div>The new secret will be added to the cluster.</div>` not `<div>The new secret will be added to the cluster.</p>`.